README.md

```diff
@@ -78,3 +37,53 @@ crontab -e
 ```
 
 *Note*: The time in adminVM is in UTC.
+
+## Special instructions
+
+### COXRAY
+
+The ETL is consist of two parts: `COXRAY_FILE` - for file upload and `COXRAY` for metadata submission.
+
+The data is available [here](https://www.kaggle.com/bachrr/covid-chest-xray) (requires Kaggle account).
+The content of archive should go into the folder `./data` (this can be changed via `COXRAY_DATA_PATH` in `coxray.py` and `coxray_file.py`) resulting in the following structure:
```

Comment:
could you mention the fact that this data must be locally downloaded manually before running the ETLs